plugin.yml

```diff
@@ -6,8 +6,6 @@ configuration:
   properties:
     skip_checkout:
       type: boolean
-    clone_url:
-      type: string
-    ref:
-      type: string
+    config:
+      type: array
```

Comment:
Have a look at [this](https://stackoverflow.com/questions/36757949/json-schema-definition-for-array-of-objects) to define a more accurate and informative schema